### ISSUE TITLE
local doc build (make html) errors out and does not complete

### ISSUE THREAD

**Issue by RCarter-IDM** [InstituteforDiseaseModeling/idmtools#2251]
**Table of Contents**

    - [Description](#description)
    - [What I Did](#what-i-did)
    - [Extras](#extras)

### Description

    - local doc build (make html) errors out and does not complete

### What I Did

```
pip install -r dev_scripts/package_requirements.txt
pip install -r docs/requirements.txt
python dev_scripts/bootstrap.py
cd docs
make html

Warning, treated as error:
autodoc: failed to import module 'create_simulation' from module 'idmtools_platform_local.internals.tasks'; the following exception was raised:
Traceback (most recent call last):
  File "c:\virt-idmtools\lib\site-packages\sphinx\ext\autodoc\importer.py", line 58, in import_module
    return importlib.import_module(modname)
  File "C:\Python38\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\github\idmtools\idmtools_platform_local\idmtools_platform_local\internals\tasks\create_simulation.py", line 26, in <module>
    class CreateSimulationTask(GenericActor):  # pragma: no cover
  File "c:\virt-idmtools\lib\site-packages\dramatiq\generic.py", line 34, in __new__
    actor_instance = actor_registry(clazz_instance, **options)
  File "c:\virt-idmtools\lib\site-packages\dramatiq\actor.py", line 225, in actor
    return decorator(fn)
  File "c:\virt-idmtools\lib\site-packages\dramatiq\actor.py", line 210, in decorator
    invalid_options = set(options) - broker.actor_options
TypeError: unsupported operand type(s) for -: 'set' and 'actor_options'

```

# Extras

Please populate system information by running `idmtools info system --issue`. To accomplish this, please follow these steps
* Ensure *idmtool-cli* packages is installed. You can do this be installing it using the command
```bash
> pip install idmtools-cli --index-url=https://packages.idmod.org/api/pypi/pypi-production/simple
```
* Run
```bash
> idmtools info system --issue
```
This will copy the system information to your clipboard
(virt-history_matching) C:\github\history_matching>idmtools info system --issue
Traceback (most recent call last):
  File "C:\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\virt-history_matching\Scripts\idmtools.exe\__main__.py", line 7, in <module>
  File "c:\virt-history_matching\lib\site-packages\idmtools_cli\main.py", line 19, in main
    cli(auto_envvar_prefix='IDMTOOLS_CLI')
  File "c:\virt-history_matching\lib\site-packages\click\core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "c:\virt-history_matching\lib\site-packages\click\core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "c:\virt-history_matching\lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\virt-history_matching\lib\site-packages\click\core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\virt-history_matching\lib\site-packages\click\core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\virt-history_matching\lib\site-packages\click\core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "c:\virt-history_matching\lib\site-packages\idmtools_cli\cli\system_info.py", line 108, in system
    raise e
  File "c:\virt-history_matching\lib\site-packages\idmtools_cli\cli\system_info.py", line 98, in system
    client = docker.from_env()
AttributeError: module 'docker' has no attribute 'from_env'

**Comment by ZDu-IDM** [comment]
Error is due to docker is not running. Better to provide a clear message.

**Comment by shchen-idmod** [comment]
Fixed in 1.7.11 release
